js/xt.js

```diff
@@ -0,0 +1,1072 @@
+                'vwap': undefined,
+                'open': this.safeString (response, 'open'),
+                'close': this.safeString (response, 'price'),
+                'last': this.safeString (response, 'price'),
+                'previousClose': undefined,
+                'change': undefined,
+                'percentage': percentage,
+                'average': this.safeString (response, 'averagePrice'),
+                'baseVolume': this.safeString (response, 'coinVol'),
+                'quoteVolume': this.safeString (response, 'moneyVol'),
+                'info': response,
+            }, market);
+        } else if (market['swap']) {
+            const ticker = this.safeValue (response, 'result', {});
+            const percentage = this.safeString (ticker, 'r');
+            const marketId = this.safeString (ticker, 's');
+            market = this.safeMarket (marketId, market, '-');
+            const symbol = market['symbol'];
+            const timestamp = this.safeString (ticker, 't');
+            return this.safeTicker ({
+                'symbol': symbol,
+                'timestamp': timestamp,
+                'datetime': this.iso8601 (timestamp),
+                'high': this.safeString (ticker, 'h'),
+                'low': this.safeString (ticker, 'l'),
+                'bid': this.safeString (ticker, 'bp'),
+                'bidVolume': undefined,
+                'ask': this.safeString (ticker, 'ap'),
+                'askVolume': undefined,
+                'vwap': undefined,
+                'open': this.safeString (ticker, 'o'),
+                'close': this.safeString (ticker, 'c'),
+                'last': this.safeString (ticker, 'c'),
+                'previousClose': undefined,
+                'change': String (this.safeNumber (ticker, 'c') - this.safeNumber (ticker, 'o')),
+                'percentage': percentage,
+                'average': undefined,
+                'baseVolume': undefined,
+                'quoteVolume': this.safeString (ticker, 'a'),
+                'info': ticker,
+            }, market);
+        }
+    }
+
+    async fetchOrderBook (symbol, limit = undefined, params = {}) {
+        /**
+         * @method
+         * @name xt#fetchOrderBook
+         * @description fetches information on open orders with bid (buy) and ask (sell) prices, volumes and other data
+         * @param {string} symbol unified symbol of the market to fetch the order book for
+         * @param {int|undefined} limit the maximum amount of order book entries to return
+         * @param {object} params extra parameters specific to the bybit api endpoint
+         * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-book-structure} indexed by market symbols
+         */
+        // TODO: Integrate Futures
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        // spot
```

Comment:
Don't know if you noticed, but usually we only leave these comments on top if it is a parser (parseTicker, orderbook, etc) otherwise, we place them below the `response = await...`. In this case we don't need to place it there either because we're not touching the result here